tokio-stream/src/wrappers/stream_close.rs

```diff
@@ -0,0 +1,59 @@
+use crate::Stream;
+use pin_project_lite::pin_project;
+use std::{
+    pin::Pin,
+    task::{Context, Poll},
+};
```

Comment:
We always use this format in Tokio.
```suggestion
use std::pin::Pin;
use std::task::{Context, Poll};
```